app/facades/trust_registry.py

```diff
@@ -151,6 +157,16 @@ async def get_did_for_actor(actor_id: str) -> List[str]:
     return [did, didcomm_invitation]
 
 
+async def get_trust_registry() -> TrustRegistry:
+    print(TRUST_REGISTRY_URL)
```

Comment:
I'd say this `print` should either be log.info or similar or left out altogether

Reply:
Woops, shouldn't be there